services-gateway-netty/src/main/java/io/scalecube/services/gateway/SessionEventHandler.java

```diff
@@ -1,13 +1,18 @@
-package io.scalecube.services.gateway.ws;
+package io.scalecube.services.gateway;
 
+import io.scalecube.services.api.ServiceMessage;
+import io.scalecube.services.gateway.ws.GatewayMessage;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public interface WebsocketGatewayHandler {
+public interface SessionEventHandler<M> {
```

Comment:
Rename it (and corresp. variables to `gatewayHandler`) plz to `GatewaySessionHandler`.